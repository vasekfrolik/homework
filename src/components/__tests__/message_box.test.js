import React from "react";
import { shallow } from "enzyme";

import MessageBox from "./../message_box";

let wrapper = shallow(<MessageBox message="No record found" />);

describe("render Message box", () => {
  it("check label and colon", () => {
    expect(wrapper.find(".message").text()).toEqual("No record found");
  });

  it("matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
