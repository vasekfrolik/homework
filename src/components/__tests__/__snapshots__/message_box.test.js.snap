// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render CheckboxSwitch matches snapshot 1`] = `ShallowWrapper {}`;

exports[`render Message box matches snapshot 1`] = `ShallowWrapper {}`;
